MANIFEST.in

```diff
@@ -10,6 +10,7 @@ recursive-include  src/pip/_vendor *LICENSE*
 recursive-include  src/pip/_vendor *COPYING*
 
 include docs/docutils.conf
+include docs/requirements.txt
```

Comment:
Previously the requirement file was not included (`prune tools`), but I guess that was actually a bug in a way. Including this feels more right to me.

Reply:
Yup, same feelings.